### DISCUSSION TITLE
Upgraded OBS, pkg installed new Scene Switcher, imported scenes gone

### DISCUSSION THREAD

**Discussion by simonpius** [WarmUpTill/SceneSwitcher#638] [Q&A]
Hi,

I want to see the scenes I previously created and imported when I updated my OBS and needed to reinstall scene switcher

OBS: 28.1.2
Scene Switcher: MACOS_Universal installed

Steps taken so far:

- Opened OBS and scene switcher through tools > advanced scene switcher
- Unticked "hide tabs which cannot be presented by macros"
- Imported old transitions file saved (.txt file)

File is active and working but I cannot see my rules like I used to so I can edit the timings, remove/add a scene etc... is there a way I can view those old transitions created by the older version?

**Comment by WarmUpTill** [comment]
Hi,

I am not sure I fully understand what the issue is.
Is the issue that not all settings were imported?
Or is the issue that you cannot find some settings at all in the UI?

In general when importing settings for the "old" tabs the sources, scenes, and transitions have to exist that are mentioned in the files otherwise the entry will be skipped.
